src/XMLimport.cpp

```diff
@@ -947,317 +964,325 @@ void XMLimport::readHostPackage( Host * pT )
         {
             if( name() == "name" )
             {
-                pT->mHostName = readElementText();
+                pHost->mHostName = readElementText();
                 continue;
             }
             else if( name() == "mInstalledModules")
             {
                 QMap<QString, QStringList> entry;
-                readMapList(entry);
+                readModulesDetailsMap(entry);
```

Comment:
Yeah, we are reading a list of Modules into a `QMap` - it is nothing to do with a "Map" being a representation of the relative location of things to one another. :man_shrugging: